### PR TITLE
Set home page logo w/title and subtitle

### PR DESCRIPTION
**Pull request by GusMeg**
## Description

Included a Dall-E made up image as our Batch 1 Logo.
Added the image to our homepage w/title and subtitle.

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #7 

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

Your ENS/address: gusmeg.eth

### CODE REVIEW

File: packages/hardhat/contracts/Registerv2.sol
Comment (carletex) [inline packages/hardhat/contracts/Registerv2.sol:1]: Let's remove this file from the PR!

**Review by carletex — CHANGES_REQUESTED** [review]
Left a tiny comment @GusMeg 

Homepage looking good :D

**Review by JollyRogerz — APPROVED** [review]
Awesome logo

File: packages/hardhat/contracts/Registerv2.sol
Comment (GusMeg) [reply packages/hardhat/contracts/Registerv2.sol:1]: Whoops... that shold've not been there. I just removed it from the branch.

**Review by carletex — APPROVED** [review]

**Pull request merged**